File: java_apx/src/test/java/net/sympower/citizen/apx/MockData.java
package net.sympower.citizen.apx;

import net.sympower.citizen.apx.dto.incomming.Quote;
import net.sympower.citizen.apx.dto.incomming.QuoteResponse;
import net.sympower.citizen.apx.dto.incomming.QuoteValue;

import java.util.List;

public class MockData {
    private static final Quote quote1 = new Quote(
            "SomeMarket", 1573599600000L, List.of(
            new QuoteValue("Order", "1"),
            new QuoteValue("Hour", "01"),
            new QuoteValue("Net Volume", "11.11"),
            new QuoteValue("Price", "11.22")
    ));

    private static final Quote quote2 = new Quote(
            "SomeMarket", 1573599600000L, List.of(
            new QuoteValue("Order", "2"),
            new QuoteValue("Hour", "02"),
            new QuoteValue("Net Volume", "22.11"),
            new QuoteValue("Price", "22.22")
    ));

    private static final Quote quote3 = new Quote(
            "OtherMarket", 1573599600000L, List.of(
            new QuoteValue("Order", "3"),
            new QuoteValue("Hour", "03"),
            new QuoteValue("Net Volume", "33.11"),
            new QuoteValue("Price", "33.22")
    ));

    public static QuoteResponse getSameQuotes() {
        return new QuoteResponse(List.of(quote1, quote2));
    }

    public static QuoteResponse getDifferentQuotes() {
        return new QuoteResponse(List.of(quote1, quote3));
    }

}

File: src/test/kotlin/net/sympower/citizen/apx/MockData.kt
package net.sympower.citizen.apx

import net.sympower.citizen.apx.dto.incomming.Quote
import net.sympower.citizen.apx.dto.incomming.QuoteResponse
import net.sympower.citizen.apx.dto.incomming.QuoteValue

object MockData {

    private val quote1 = Quote(
        "SomeMarket", 1573599600000, listOf(
            QuoteValue("Order", "1"),
            QuoteValue("Hour", "01"),
            QuoteValue("Net Volume", "11.11"),
            QuoteValue("Price", "11.22")
        )
    )

    private val quote2 = Quote(
        "SomeMarket", 1573599600000, listOf(
            QuoteValue("Order", "2"),
            QuoteValue("Hour", "02"),
            QuoteValue("Net Volume", "22.11"),
            QuoteValue("Price", "22.22")
        )
    )

    private val quote3 = Quote(
        "OtherMarket", 1573599600000, listOf(
            QuoteValue("Order", "3"),
            QuoteValue("Hour", "03"),
            QuoteValue("Net Volume", "33.11"),
            QuoteValue("Price", "33.22")
        )
    )

    fun getSameQuotes(): QuoteResponse =
        QuoteResponse(listOf(quote1, quote2))

    fun getDifferentQuotes(): QuoteResponse =
        QuoteResponse(listOf(quote1, quote3))

}